specifyweb/specify/migrations/0001_initial.py

```diff
@@ -5762,6 +5762,36 @@ class Migration(migrations.Migration):
                 'ordering': (),
             },
         ),
+        migrations.CreateModel(
+            name='AutonumschColl',
+            fields=[
+                ('collectionid', models.ForeignKey(db_column='CollectionID', on_delete=django.db.models.deletion.DO_NOTHING, primary_key=True, serialize=False, to='specify.collection')),
+            ],
+            options={
+                'db_table': 'autonumsch_coll',
+                'managed': False,
+            },
+        ),
+        migrations.CreateModel(
+            name='AutonumschDiv',
+            fields=[
+                ('divisionid', models.ForeignKey(db_column='DivisionID', on_delete=django.db.models.deletion.DO_NOTHING, primary_key=True, serialize=False, to='specify.division')),
+            ],
+            options={
+                'db_table': 'autonumsch_div',
+                'managed': False,
+            },
+        ),
+        migrations.CreateModel(
+            name='AutonumschDsp',
+            fields=[
+                ('disciplineid', models.ForeignKey(db_column='DisciplineID', on_delete=django.db.models.deletion.DO_NOTHING, primary_key=True, serialize=False, to='specify.discipline')),
+            ],
+            options={
+                'db_table': 'autonumsch_dsp',
+                'managed': False,
+            },
+        ),
```

Comment:
Are we keeping these in the initial migration? 
If so, my previous comments still apply: https://github.com/specify/specify7/pull/7455#discussion_r2543046732, https://github.com/specify/specify7/pull/7455#discussion_r2543051371